web/src/routes/OffersPage.tsx

```diff
@@ -64,13 +64,13 @@ const OffersPage = ({ city, cityCode, languageCode }: CityRouteProps): ReactElem
       code,
     }
   })
-
+  const pageTitle = `${t('pageTitle')} - ${city.name}`
```

Comment:
the results are fine imo. When a mail client is detecting rtl language it will auto turn direction f.e.
<img width="1503" alt="image" src="https://github.com/digitalfabrik/integreat-app/assets/37902063/9ee74942-34ad-41f9-9de5-f7d3b75bed79">
But i think thats fine